pyhht/visualization.py

```diff
@@ -72,7 +73,8 @@ def plot_imfs(signal, imfs, time_samples=None, fignum=None, show=True):
     ax.tick_params(which='both', left=False, bottom=False, labelleft=False,
                    labelbottom=False)
     ax.grid(False)
-    ax.set_ylabel('Signal')
+    ax.set_ylabel('Signal',rotation=0)
```

Comment:
Please expose the `rotation` parameter in the function definition so others can use it as they like. Keep the default as 0. Also please run `flake8` on the file and fix all related issues.